airbyte-integrations/connectors/source-pinterest/source_pinterest/schemas/campaign_analytics.json

```diff
@@ -7,7 +7,7 @@
       "format": "date"
     },
     "AD_ACCOUNT_ID": {
-      "type": ["null", "integer"]
+      "type": ["null", "string", "number"]
```

Comment:
and here is the same, change it `["string"]` please, and then we will run the tests again, it should fix the problem we had with failed tests

Reply:
sweet, it's changed